### ISSUE TITLE
flyingcloud -h / --help should display something without asking sudo

### ISSUE THREAD

**Issue by arthurzenika** [cookbrite/flyingcloud#21]
```
$ flyingcloud -h
[sudo] password for arthur: 
ctrl-C
```
I can understand that flyingcloud needs to sudo at some point, but you should be able to read a --help message without sudo, and display some information before sudo is run to warn the user why you need to be root.

**Comment by georgevreilly** [comment]
Good point. It should be possible for the `flyingcloud` script to peek at `$*` and not `sudo` if `-h` or `--help` is present.
